client/interfaces.go

```diff
@@ -359,7 +361,7 @@ type InstanceServer interface {
 	GetStoragePoolVolumesWithFilterAllProjects(pool string, filters []string) (volumes []api.StorageVolume, err error)
 	GetStoragePoolVolume(pool string, volType string, name string) (volume *api.StorageVolume, ETag string, err error)
 	GetStoragePoolVolumeState(pool string, volType string, name string) (state *api.StorageVolumeState, err error)
-	CreateStoragePoolVolume(pool string, volume api.StorageVolumesPost) (err error)
+	CreateStoragePoolVolume(pool string, volume api.StorageVolumesPost) (op Operation, err error)
```

Comment:
This change shouldn't be needed, so please can you explain it?
Also, we cannot do it anyway, as it would break the existing API users.